app/assets/javascripts/application.js

```diff
@@ -1144,3 +1144,33 @@ function appendOptionsToDropdown(options, selectElement) {
     });
     selectElement.append(optionsArray);
 }
+
```

Comment:
I think it's cleaner (and preferred in standard js libraries) to consolidate these two methods into a single setElementsEnabled(boolean) method.  Even if it doesn't save lines of code in the implementation, it makes the API simpler to remember and the do/undo connection between the two even more explicit

Reply:
Since I have other refactoring to do I'll fold this in as well.

Reply:
ef48f0b